.coveragerc

```diff
@@ -15,6 +15,6 @@ source=
 precision = 2
 ignore_errors = True
 exclude_lines =
-    if TYPE_CHECKING
+    if (typing\.)?TYPE_CHECKING
```

Comment:
Seems unnecessary if we consistently import from `typing` instead of typing itself, which so far seems the preferred practice (to me, anyway).